python/astshim/test.py

```diff
@@ -131,3 +132,66 @@ def checkBasicSimplify(self, amap):
             self.assertEqual(mb.getNout(), cmp3.getNout())
             self.assertEqual(cmp3.getNin(), cmp3simp.getNin())
             self.assertEqual(cmp3.getNout(), cmp3simp.getNout())
+
+
+def makeCoeffs(nIn, nOut):
```

Comment:
Please make the name more specific and (unless you see a use case) mark it private by giving the name a leading underscore. I'm thinking of something like `_makePolyMapCoeffs`. Perhaps I should have done this when it was in a test file, but I think it matters more now that it is in a library module.